docs/Flare/README.md

```diff
@@ -0,0 +1,49 @@
+# Flare
```

Comment:
I suppose we decided not to use inner layout for paddings?

Reply:
No, I've added the `insetPaddingMode` prop for that